### PR TITLE
`-m` regex test is deprecated. Use `--match-test` instead

### PR DESCRIPTION
**Pull request by AAYUSH-GUPTA-coder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1411,6 +1411,17 @@ ANYWHERE written in plain text
 
 ## Forked Tests
 *[⌨️ (25:47) | Forked Tests ](https://youtu.be/sas02qSFZ74?t=1547)*
+
+- The test option `-m` regex is deprecated. Please use `--match-test` instead.
```
Comment (PatrickAlphaC) [inline README.md:1415]:
```suggestion
- The test option `-m` regex is deprecated. Please use `--match-test` or `--mt` instead.
```

**Comment by PatrickAlphaC** [comment]
ty!

**Comment by AAYUSH-GUPTA-coder** [comment]
Thank you, sir, for all the Amazing Work and providing free, Quality Education.

**Comment by AAYUSH-GUPTA-coder** [comment]
Thank you, ser, for all the Amazing Work and for giving Free, Quality Education.

**Pull request merged**